morello/morello.md

```diff
@@ -117,87 +110,76 @@ reserved.
 The following support level definitions are used by the ACLE
 specifications:
 
-**Release**
+_**Release**_
 
-:   Arm considers this specification to have enough implementations,
-    which have received sufficient testing, to verify that it is
-    correct. The details of these criteria are dependent on the scale
-    and complexity of the change over previous versions: small, simple
-    changes might only require one implementation, but more complex
-    changes require multiple independent implementations, which have
-    been rigorously tested for cross-compatibility. Arm anticipates that
-    future changes to this specification will be limited to
-    typographical corrections, clarifications and compatible extensions.
+|   Arm considers this specification to have enough implementations,
+|   which have received sufficient testing, to verify that it is
+|   correct. The details of these criteria are dependent on the scale
+|   and complexity of the change over previous versions: small, simple
+|   changes might only require one implementation, but more complex
+|   changes require multiple independent implementations, which have
+|   been rigorously tested for cross-compatibility. Arm anticipates that
+|   future changes to this specification will be limited to
+|   typographical corrections, clarifications and compatible extensions.
```

Comment:
Are the `|    ` needed? They show up as a single `|` in the rendering at https://github.com/ValeriaDaneva/acle/blob/github.io-morello/morello/morello.md#change-control and they seems useless. Please remove.

Please remove also al cases below.